packages/volto/src/components/manage/Toolbar/Toolbar.jsx

```diff
@@ -314,6 +314,9 @@ class Toolbar extends Component {
   };
 
   handleClickOutside = (e) => {
+    if (e.target.tagName==='svg'){
+      return;
```

Comment:
This does not pass ESLint check. You need to fix this. See [Code quality](https://6.docs.plone.org/volto/contributing/index.html#code-quality)